libs/ui/catalog/src/lib/organisations-sort/organisations-sort.component.html

```diff
@@ -0,0 +1,19 @@
+<div
+  class="flex flex-wrap sm:flex-nowrap bg-primary text-white px-6 py-7 mt-4 mb-8 rounded-lg"
+>
+  <span class="flex-grow mb-4 mr-4 sm:mr-16">
+    <p translate>organisation.sort.intro</p>
```

Comment:
the `p` element has a bottom margin which makes the box look weird with longer text; using `m-0` should fix it

Reply:
I can't reproduce this and don't have any margin on `<p>` due to tailwind. I'll add a `sm:mb-0` to the <span> above it, though, since it makes the margin grow too much. Maybe this is what you mean?